src/PuliPlugin.php

```diff
@@ -466,10 +466,10 @@ private function loadPuliPackages()
         $packages = array();
 
         $output = $this->puliRunner->run(
-            'package --list --format \'%name%;%installer%;%install_path%;%state%\''
+            'package --list --format "%name%;%installer%;%install_path%;%state%"'
```

Comment:
The second problem is not a Puli one but a Symfony one. In PuliRunner class, at line 81 ` $process = new Process($fullCommand);` I add just after `$process->setEnhanceWindowsCompatibility(false);` and it run like a charm on Windows with single quotes. Think it cause by `ProcessUtils::escapeArgument`.

I'll let you know want I get a nice fix about it.